google-cloud-pubsub/lib/google/cloud/pubsub/service.rb

```diff
@@ -38,65 +36,58 @@ class Service
 
         ##
         # Creates a new Service instance.
-        def initialize project, credentials, host: nil, timeout: nil,
-                       client_config: nil
+        def initialize project, credentials, host: nil, timeout: nil
           @project = project
           @credentials = credentials
-          @host = host || V1::PublisherClient::SERVICE_ADDRESS
+          @host = host
           @timeout = timeout
-          @client_config = client_config || {}
           @client_id = SecureRandom.uuid.freeze
         end
 
-        def channel
-          require "grpc"
-          GRPC::Core::Channel.new host, chan_args, chan_creds
-        end
-
-        def chan_args
-          { "grpc.max_send_message_length"           => -1,
-            "grpc.max_receive_message_length"        => -1,
-            "grpc.keepalive_time_ms"                 => 300_000,
```

Comment:
Are these grpc args necessary for pubsub? If so, we probably need to set them somewhere else.

Reply:
Ah, thanks!